src/EnergyPlus/DaylightingManager.cc

```diff
@@ -4371,543 +4345,663 @@ namespace DaylightingManager {
+		int zoneIndex = 0;
+		for (auto & daylCntrl : ZoneDaylight) {
+			zoneIndex++;
+			if ( daylCntrl.TotalDaylRefPoints > 0 ){
+				auto & zone( Zone(zoneIndex) );
+
+				// Calc cos and sin of Zone Relative North values for later use in transforming Reference Point coordinates
+				CosZoneRelNorth = std::cos( -zone.RelNorth * DegToRadians );
+				SinZoneRelNorth = std::sin( -zone.RelNorth * DegToRadians );
+
+				rLightLevel = GetDesignLightingLevelForZone( zoneIndex );
+				CheckLightsReplaceableMinMaxForZone( zoneIndex );
+
+				for ( refPtNum = 1; refPtNum <= daylCntrl.TotalDaylRefPoints; ++refPtNum ){
+					auto & curRefPt( DaylRefPt( daylCntrl.DaylRefPtNum( refPtNum ) ) ); // get the active daylighting:referencepoint
+					curRefPt.indexToFracAndIllum = refPtNum; // back reference to the index to the ZoneDaylight structure arrays related to reference points
+					if ( DaylRefWorldCoordSystem ) {
+						//transform only by appendix G rotation
+						daylCntrl.DaylRefPtAbsCoord( 1, refPtNum ) = curRefPt.x * CosBldgRotAppGonly - curRefPt.y * SinBldgRotAppGonly;
+						daylCntrl.DaylRefPtAbsCoord( 2, refPtNum ) = curRefPt.x * SinBldgRotAppGonly + curRefPt.y * CosBldgRotAppGonly;
+						daylCntrl.DaylRefPtAbsCoord( 3, refPtNum ) = curRefPt.z;
+					} else {
+						//Transform reference point coordinates into building coordinate system
+						Xb = curRefPt.x * CosZoneRelNorth - curRefPt.y  * SinZoneRelNorth + zone.OriginX;
+						Yb = curRefPt.x * SinZoneRelNorth + curRefPt.y  * CosZoneRelNorth + zone.OriginY;
+						//Transform into World Coordinate System
+						daylCntrl.DaylRefPtAbsCoord( 1, refPtNum ) = Xb * CosBldgRelNorth - Yb * SinBldgRelNorth;
+						daylCntrl.DaylRefPtAbsCoord( 2, refPtNum ) = Xb * SinBldgRelNorth + Yb * CosBldgRelNorth;
+						daylCntrl.DaylRefPtAbsCoord( 3, refPtNum ) = curRefPt.z + zone.OriginZ;
+						if ( doTransform ) {
+							Xo = daylCntrl.DaylRefPtAbsCoord( 1, refPtNum ); // world coordinates.... shifted by relative north angle...
+							Yo = daylCntrl.DaylRefPtAbsCoord( 2, refPtNum );
+							// next derotate the building
+							XnoRot = Xo * CosBldgRelNorth + Yo * SinBldgRelNorth;
+							YnoRot = Yo * CosBldgRelNorth - Xo * SinBldgRelNorth;
+							// translate
+							Xtrans = XnoRot * std::sqrt( NewAspectRatio / OldAspectRatio );
+							Ytrans = YnoRot * std::sqrt( OldAspectRatio / NewAspectRatio );
+							// rerotate
+							daylCntrl.DaylRefPtAbsCoord( 1, refPtNum ) = Xtrans * CosBldgRelNorth - Ytrans * SinBldgRelNorth;
+							daylCntrl.DaylRefPtAbsCoord( 2, refPtNum ) = Xtrans * SinBldgRelNorth + Ytrans * CosBldgRelNorth;
+						}
+					}
+					refName = curRefPt.Name;
+					PreDefTableEntry( pdchDyLtZone, refName, daylCntrl.Name );
+					PreDefTableEntry( pdchDyLtKind, refName, "Detailed" );
+					// (1=continuous, 2=stepped, 3=continuous/off)
+					if ( daylCntrl.LightControlType == Continuous ) {
+						PreDefTableEntry( pdchDyLtCtrl, r
```

Comment:
I feel like maybe GH got confused too because of the size of the diff on this PR.  The code I'm referring to is the [`doesDayLightingUseDElight`](https://github.com/NREL/EnergyPlus/blob/eed35b53bcd20c03281ff34c50a202d99505552b/src/EnergyPlus/DaylightingManager.cc#L4994-L5002) function, where the for-loop doesn't have braces:

```
	bool
	doesDayLightingUseDElight()
	{
		for (auto & znDayl : ZoneDaylight )
			if ( znDayl.DaylightMethod == DElightDaylighting ){
				return true;
			}
		return false;
	}
```